selfapi.js

```diff
@@ -555,13 +593,21 @@ function getHandlerExporter (app) {
   // `app` is an express-like server app.
   return function (method, path, parameters) {
     // Support restify.
-    if (method === 'delete' && ('del' in app)) {
-      method = 'del';
+    if (method === 'del' && ('delete' in app)) {
+      method = 'delete';
     }
     app[method](path, parameters.handler);
   };
 }
 
+// Stringify object, pass down strings
+function jsonStringify(strOrObject) {
```

Comment:
Bonus super-nits:
- Please add a space between `jsonStringify` and `(` to respect local coding style (this project needs a linter)
- The variable name `strOrObject` looks a bit convoluted. What about simply calling it `value`? It's pretty obvious what it contains if you look at the function body or its comment.